client/state/utils/keyed-reducer.js

```diff
@@ -100,37 +82,9 @@ export const keyedReducer = ( keyPath, reducer ) => {
 		);
 	}
 
-	const persistingReducer = setupReducerPersistence( reducer );
-
-	const initialState = persistingReducer( undefined, { type: '@@calypso/INIT' } );
+	const initialState = reducer( undefined, { type: '@@calypso/INIT' } );
 
 	const combinedReducer = ( state = {}, action ) => {
-		if ( action.type === SERIALIZE ) {
```

Comment:
Let's also update the inline docs on L30 above to no longer mention the old serialization action types.